functions/__fzf_display_value_or_error.fish

```diff
@@ -1,10 +1,12 @@
 # helper function for __fzf_search_shell_variables
-function __fzf_display_value_or_error --argument-names variable_name --description "Displays either the value of the variable passed in, or an informative message if it is not available."
-    if set --query $variable_name
-        echo $$variable_name
+function __fzf_display_value_or_error --argument-names variable_name variable_file --description "Displays either the value of the variable passed in, or an informative message if it is not available."
+    set matches (cat $variable_file | string match --regex "^\\\$$variable_name(?::|\[).+")
```

Comment:
@kidonng [UUOC](https://en.wikipedia.org/wiki/Cat_(Unix)#Useless_use_of_cat). This will do it:

```fish
string match --regex "^\\\$$variable_name(?::|\[).+" <$variable_file
```

Reply:
Thanks for helping me review this, Jorge! Please also chime in on the feature design; I know you have a good eye for these things.